### PR TITLE
Fix 'not available' image loading as a jpg

### PR DESCRIPTION
**Pull request by josephspens**
We had previously loaded the _not available_ image as an SVG in the same manner we load the map SVG, the client library handles it gracefully and ignores all methods run against it since the manifest doesn't load. It's in effect a no-op instance in which you can pan and zoom on the not available image.

This no longer works as we're loading a JPG instead of an SVG. We need to break flow in the map fetch if the fetch fails, and instead render an `img`.

### CODE REVIEW

File: src/ticketmap/index.tsx [hunk 1]
```diff
@@ -117,11 +125,9 @@ export default class TicketMap extends Component<Props & DefaultProps, State> {
   }
 
   async fetchMap() {
-    const mapSvgUrl = `${this.configFilePath}/map.svg`
-    const mapNotAvailableUrl = 'https://maps.ticketevolution.com/maps/not_available.jpg' // Only exists on prod
-    let mapResponse = await fetch(mapSvgUrl)
+    const mapResponse = await fetch(`${this.configFilePath}/map.svg`)
     if (!mapResponse.ok) {
-      mapResponse = await fetch(mapNotAvailableUrl)
+      throw new MapNotFoundError('This map is not currently available.');
```
Comment (mayavera) [inline src/ticketmap/index.tsx]: The `message` for `MapNotFoundError` seems to be constant. We could set this as the default for its constructor.

```ts
class MapNotFoundError extends Error {
  constructor(message = 'This map is not currently available.') {
    super(message)
  }
}}
```

File: src/ticketmap/index.tsx [hunk 2]
```diff
@@ -76,7 +80,8 @@ export default class TicketMap extends Component<Props & DefaultProps, State> {
       tooltipZoneId: '',
       tooltipX: 0,
       tooltipY: 0,
-      ticketGroups: this.props.ticketGroups
+      ticketGroups: this.props.ticketGroups,
+      displayNotFoundImage: false
```
Comment (mayavera) [inline src/ticketmap/index.tsx]: I'd like to keep our state properties from becoming view modifiers. `displayNotFoundImage` reads as a sort of manual control of a view element. I would prefer if we renamed it to some description of the app's _state_, such as `mapFetchFailed`. `render()` then has the ability to generate whatever view seems necessary (in this case, creating a not found image).

```suggestion
      mapFetchFailed: false
```

Comment (josephspens) [reply src/ticketmap/index.tsx]: I changed it to `mapNotFound` to correlate with the error

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ticket-evolution-seatmaps",
-  "version": "0.5.4",
+  "version": "0.5.6",
```
Comment (mayavera) [inline package.json]: What happened to `0.5.5`?

Comment (josephspens) [reply package.json]: I think I forgot to update the package.json for the last release.

File: src/ticketmap/index.tsx [hunk 4]
```diff
@@ -42,6 +42,13 @@ interface ElementProperties {
 
 type PropertiesForElement = (element: Element) => ElementProperties
 
+class MapNotFoundError extends Error {
+  name = 'MapNotFoundError'
+  constructor(message = 'This map is not currently available.') {
+    super(message)
```
Comment (Pharserror) [inline src/ticketmap/index.tsx]: I'm like John Travolta in Pulp Fiction looking for these `;`'s.

Comment (josephspens) [reply src/ticketmap/index.tsx]: FYI this project enforces no semicolons

Comment (mayavera) [reply src/ticketmap/index.tsx]: This project follows [StandardJS](https://standardjs.com/), which advises agains semicolons

Comment (Pharserror) [reply src/ticketmap/index.tsx]: 😢

**Pull request merged**